generator/subaru/subaru_global_2017.dbc

```diff
@@ -0,0 +1,59 @@
+CM_ "IMPORT _subaru_global_2017.dbc"
```

Comment:
I'd rename this to `_subara_global.dbc` since `subaru_global_2017.dbc` and `subaru_global_2020_hybrid.dbc` both import it

Reply:
I renamed the template to _subaru_global.dbc and reran generator